example-client/.env.client.example

```diff
@@ -0,0 +1,38 @@
+# Copy this file to a new file named '.env' to automatically load these
+# values as environment variables when running the client.
+# You will need to fill in the values based on which scenario you are
+# running against:
+# 1. A client testing against a Structure registered locally in the 
+#    Skatepark emulator.
+# 2. A client running against a Structure hosted in Griptape Cloud.
+
+# If the Structure is running in the Skatepark emulator, this value
+# will be the ID returned when building or registering the Structure
+# from the CLI.
+# In Griptape Cloud, it will be the ID of the structure as found on
+# https://cloud.griptape.ai/structures .
+GT_STRUCTURE_ID=YOUR_STRUCTURE_ID_HERE
+
+# If running against the Skatepark emulator, you will need to set
+# this environment variable (or put it in a .env file) to tell the
+# client which URL Skatepark is listening on. The default value is
+# http://127.0.0.1:5000 .
+# For a Structure in Griptape Cloud, this environment variable 
+# should be set to https://cloud.griptape.ai/ .
+GT_CLOUD_BASE_URL=TARGET_URL_HERE
+
+# The Skatepark emulator does NOT require a Griptape API key.
+# However, Griptape Cloud requires a valid API Key in order
+# to authorize calling the API. You can generate a key by
+# visiting https://cloud.griptape.ai/keys
+GT_CLOUD_API_KEY=YOUR_API_KEY_HERE
+
+# When running in Griptape Cloud, the Structure first reports
+# a status of QUEUED before transitioning to RUNNING. Skatepark
+# emulates this behavior by applying a default time to remain 
+# in the QUEUED state in order to allow your client to 
+# handle this state appropriately.
+# You can override the time that the Structure remains in
+# the QUEUED state within Skatepark by setting the environment
+# variable GT_SKATEPARK_QUEUE_DELAY to the desired time in seconds.
+# GT_SKATEPARK_QUEUE_DELAY=VALUE_IN_SECONDS_TO_QUEUE
```

Comment:
I think we should provide a valid default, even in the example. I.e. `GT_SKATEPARK_QUEUE_DELAY=2`

Reply:
bleh I don't agree but I don't disagree strongly enough to care. Updated.